agendas/2023/01-Jan/05-wg-primary.md

```diff
@@ -120,7 +121,15 @@ hold additional secondary meetings later in the month.
    - [Ready for review](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Ready+for+review+%F0%9F%99%8C%22+sort%3Aupdated-desc)
    - [All open action items (by last update)](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Action+item+%3Aclapper%3A%22+sort%3Aupdated-desc)
    - [All open action items (by meeting)](https://github.com/graphql/graphql-wg/projects?query=is%3Aopen+sort%3Aname-asc)
-1. Updates on [Fragment Arguments](https://github.com/graphql/graphql-js/pull/3152) (15m, Matt)
+1. Review of pending approved PRs (5m, Roman), making this regular part of agenda. PRs ready to go: 
+    [974](https://github.com/graphql/graphql-spec/pull/974), 
+    [975](https://github.com/graphql/graphql-spec/pull/975), 
+    [979](https://github.com/graphql/graphql-spec/pull/979),
+    [981](https://github.com/graphql/graphql-spec/pull/981); 
+    Calling for action to: 
+      [983](https://github.com/graphql/graphql-spec/pull/983),
+      [984](https://github.com/graphql/graphql-spec/pull/984),
+      [986](https://github.com/graphql/graphql-spec/pull/986)
```

Comment:
You appear to have deleted Matt's agenda item, please restore it. I don't think 5 minutes is nearly enough time to cover 7 pull requests - suggest bumping to _at the very least_ 15 minutes, 30 is probably more appropriate. Please conform to the markdown formatting used for the other agenda items (i.e. use bullets):

```suggestion
1. Updates on [Fragment Arguments](https://github.com/graphql/graphql-js/pull/3152) (15m, Matt)
1. Review of pending approved PRs (15m, Roman)
   - making this regular part of agenda
   - PRs ready to go: 
     - [974](https://github.com/graphql/graphql-spec/pull/974), 
     - [975](https://github.com/graphql/graphql-spec/pull/975), 
     - [979](https://github.com/graphql/graphql-spec/pull/979),
     - [981](https://github.com/graphql/graphql-spec/pull/981); 
   - Calling for action to: 
     - [983](https://github.com/graphql/graphql-spec/pull/983),
     - [984](https://github.com/graphql/graphql-spec/pull/984),
     - [986](https://github.com/graphql/graphql-spec/pull/986)
```